doc/tutorial.rst

```diff
@@ -0,0 +1,112 @@
+Tutorial
+########
+
+Below is a basic tutorial on how to operate the EMVA1288 module.
+
+Description
+===========
+
+If you are in a rush there is one utility class that allows to do the
+full process in a very simple way
+
+::
+
+    from emva1288 import Emva1288
+
+    # Load your data descriptor file
+    e = Emva1288(filename)
+    # Print the results
+    e.results()
+    # Plot the graphics
+    e.plot()
+
+If you are interested keep reading.
+
+The code is divided in 7 parts
```

Comment:
I suggest that it stays like this for the moment and treat the camera simulator and dataset generator as add-ons to the main process module.